### PR TITLE
Terragrunt support implemented

### PR DESCRIPTION
**Pull request by Shokubeni**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,47 @@
...
-    secret_key           = "<secret_key>"
-    bucket               = "<bucket_name>"
-    region               = "eu-west-1"
-    workspace_key_prefix = "environment"
-    key                  = "state"
-  }
-}
-```
+## Requirements
+We use Terragrunt. It is provides extra tools for working with multiple Terraform modules. In the beginning you need to install Terraform and Terragrunt console applications as showing below.
```
Comment (NikitaZhevnitskiy) [inline README.md]: At the beginning

File: README.md [hunk 2]
```diff
@@ -1,40 +1,47 @@
...
-private_subnets = {
-  "172.16.0.0/20"  = "eu-west-1b"
-}
-public_subnets = {
-  "172.16.16.0/20" = "eu-west-1b"
-}
-```
\ No newline at end of file
+## Quick start
+In the beggining you must create `cluster_operator` profile inside `~/.aws/creditnails` file or use another
```
Comment (NikitaZhevnitskiy) [inline README.md]: At the beginning

Comment (Shokubeni) [reply README.md]: > At the beginning

fixed

File: README.md (hunk 1 above)
Comment (Shokubeni) [reply README.md]: > At the beginning

fixed

**Pull request merged**